monitoring/scripts/comprehensive_test.py

```
@@ -0,0 +1,752 @@
+                response = requests.get('http://localhost:5001/alerts/summary', timeout=10)
+                if response.status_code == 200:
+                    data = response.json()
+                    total_alerts = data.get('total_alerts', 0)
+                    if total_alerts > 0:
+                        results['notification_delivery'] = True
+                        logger.info(f"✅ Notifications being delivered: {total_alerts} alerts")
+                    else:
+                        logger.info("ℹ️  No active alerts (this is normal)")
+                else:
+                    logger.warning(f"⚠️  Cannot check notification delivery: {response.status_code}")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)